pkg/controllers/osc/osc_controller.go

```diff
@@ -267,6 +267,9 @@ func (r *Reconciler) reconcileOperatingSystemConfigs(ctx context.Context, md *cl
 		r.containerRuntimeConfig,
 		r.kubeletFeatureGates,
 	)
+	if err != nil {
```

Comment:
This check exists [here](https://github.com/kubermatic/operating-system-manager/blob/main/pkg/controllers/osc/osc_controller.go#L277-L279) but it seems like the code to [generate edge script](https://github.com/kubermatic/operating-system-manager/blob/main/pkg/controllers/osc/osc_controller.go#L271-L275) was mistakenly added before the null check.

Can you please remove https://github.com/kubermatic/operating-system-manager/blob/main/pkg/controllers/osc/osc_controller.go#L277-L279 so that we have 1 nil check instead of 2.